src/EnergyPlus/Coils/CoilCoolingDXCurveFitPerformance.cc

```diff
@@ -583,6 +583,19 @@ void CoilCoolingDXCurveFitPerformance::calcStandardRatings210240(EnergyPlus::Ene
             this->standardRatingIEER += IEERWeightingFactor[RedCapNum] * EERReduced;
         }
 
+        // IEER 2022 -->
+        // TODO: we can always decide and give precedence to Alternate Mode 1 or Alternate Mode 2 if present | Needs Discussion about the
+        // applicability.
+        Real64 IEER_2022(0.0);
+        Real64 EER_2022(0.0);
+        Real64 NetCoolingCapRated2022(0.0);
+        std::tie(IEER_2022, NetCoolingCapRated2022, EER_2022) =
+            StandardRatings::IEERCalulcationCurveFit(state, "Coil:Cooling:DX:CurveFit", this->normalMode);
+
+        this->standardRatingIEER2 = IEER_2022;
+        this->standardRatingEER2 = EER_2022;
```

Comment:
But I see you then just assign the temporary tied variables back to member variables on the class.  Why create the temporary variables at all?

Reply:
Removed the temporary variables.